src/pages/how-does-react-tell-a-class-from-a-function/index.zh-hant.md

```diff
@@ -0,0 +1,553 @@
+---
+title: React 是如何從 Function 中區別出 Class？
```

Comment:
```suggestion
title: React 如何區分函式（Function）和類別（Class）？
```
Since you use `函式` for function below, let's also use Chinese as the primary language here?

Reply:
OK!